### PR TITLE
Removing double imports found by staticcheck

### PR DESCRIPTION
**Pull request by ytimocin**
# Description
Removing double imports found by staticcheck

## Type of change
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

### CODE REVIEW

**Review by willdavsmith — APPROVED** [review]

**Review by lakshmimsft — APPROVED** [review]

File: pkg/cli/cmd/env/update/update_test.go [hunk 1]
```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/radius-project/radius/pkg/cli/framework"
 	"github.com/radius-project/radius/pkg/cli/output"
 	"github.com/radius-project/radius/pkg/cli/workspaces"
-	"github.com/radius-project/radius/pkg/corerp/api/v20231001preview"
```
Comment (rynowak) [inline pkg/cli/cmd/env/update/update_test.go:33]: Is this something we could turn on in `golanglint-ci`? I think this is a good change, I'm just wondering how we could automate it.

**Review by rynowak — APPROVED** [review]

File: pkg/cli/cmd/env/update/update_test.go (hunk 1 above)
Comment (ytimocin) [reply pkg/cli/cmd/env/update/update_test.go:33]: I found it by using `staticcheck ./...`. We can add that to a CI/CD we have.

Comment (rynowak) [reply pkg/cli/cmd/env/update/update_test.go:33]: I did some research, it sounds like we should add staticcheck. I thought it was bundled into golanglint-ci but it isn't exactly, sounds like there is a history there.

Comment (ytimocin) [reply pkg/cli/cmd/env/update/update_test.go:33]: Created this feature request: https://github.com/radius-project/radius/issues/7795. cc/ @rynowak

**Pull request merged**